app/views/posts/update.js.erb

```diff
@@ -1,7 +1,9 @@
 <% if @post.valid? %>
-  var postElement = $("#post_<%= @post.id %>").get(0);
+  var $postElement = $("#post_<%= @post.id %>");
+  var postElement = $postElement.get(0);
   var html = "<%= j post_preview(@post, fit: :compact, show_deleted: true, show_votes: @show_votes, size: @preview_size) %>";
   Alpine.morph(postElement, html);
+  $postElement.removeData();
```

Comment:
this clears jQuery's internal data cache, otherwise it would load stale tags value when, for example, reopening the edit mode for the same post.